test/test_helper.rb

```diff
@@ -9,6 +9,12 @@
   STDERR.puts 'Running without SimpleCov'
 end
 
+begin
```

Comment:
If you just require `pry`, it will load all pry plugins.  This will lead to less churn if/when we add more pry configurations.  I won't block if this doesn't seem interesting to you.  It would require adding `pry` to the gemfile, then we could remove the `rescue` here since pry is all platforms